core/components/atoms/image/image.js

```diff
@@ -5,14 +5,14 @@ import PropTypes from 'prop-types'
 const Image = props => <Image.Element src={props.source} alt={props.alt} {...props} />
 
 Image.Element = styled.img`
-  user-select: none;
-  pointer-events: none;
+  height: auto;
+  max-width: 100%;
 `
 
 Image.propTypes = {
   /** URL of the image */
   source: PropTypes.string.isRequired,
-  /** Alt info for the image */
+  /** Alt text is mandatory, please read the accessibility section  */
   alt: PropTypes.string
```

Comment:
The way to make something mandatory with proptypes is to mark it as required. This will show up a warning on dev tools when it's missing

```js
alt: PropTypes.string.isRequired
```

Ref: https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
❤️ thanks Sid